runbotics-orchestrator/src/main/java/com/runbotics/web/rest/ProcessResource.java

```diff
@@ -261,16 +261,16 @@ public ResponseEntity<ProcessDTO> getProcess(@PathVariable Long id) {
 
     /**
      * {@code GET  /processes/name/:processName/is-available} : check if given process name is available
-     * 
+     *
      * @param processName the prcess name to check if available
      * @return the {@link ResponseEntity} with status {@code 200 (OK)} or with status {@code 409 (Conflict)}.
      */
-    @PreAuthorize("@securityService.checkFeatureKeyAccess('" + FeatureKeyConstants.PROCESS_READ + "')")
+    @PreAuthorize("@securityService.checkFeatureKeyAccess('" + FeatureKeyConstants.PROCESS_LIST_READ + "')")
```

Comment:
hmm, imo this shouldnt be list read here, it is used in process creation

in the perfect world it would PROCESS_READ or PROCESS_ADD